### DISCUSSION TITLE
Have any way to avoid executing `start_kgdb.py` as root?

### DISCUSSION THREAD

**Discussion by Squirre17** [0xricksanchez/like-dbg#142] [Q&A] [answered]
if I execute `./start_kgdb.py` normally. It will be failed due to not have permission which docker need.
but if I use  `sudo ./start_kgdb.py`, I will lose all my venv and tmux session as following.
```
-> No active virtual environment found!                                    
-> Not running inside a TMUX session!                                      
(.like-dbg) 
``` 
That's meanings everything need to do as super-user from the beginning?

**Comment by 0xricksanchez** [comment, marked as answer]
Using `sudo` should not be necessary if your normal user can run docker containers.
Can you run `docker run hello-world` without `sudo`?
If not, this may be for you: https://docs.docker.com/engine/install/linux-postinstall/
